index.html

```diff
@@ -1,24 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./index.css" />
-    <title>Awesome Books</title>
-  </head>
-  <body>
-    <header>
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://cdn.jsdelivr.net/npm/luxon@2.0.2/build/global/luxon.min.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="index.css">
+ <title> Awesome Books</title>
+</head>
+<body>
+  <main class="main-container">
+    <div class="menu">
+      <nav class="navbar-container">
+        <a class="logo"><span>Awesome Books</span></a>
+        <ul class="nav-list">
+          <li id="add-list" class="nav-link">List</li>
+          <li id="add-new" class="nav-link">Add New</li>
+          <li id="contact" class="nav-link">Contact</li>
+        </ul>
+        <div class="hamburger">
+          <div class="line1"></div>
+          <div class="line2"></div>
+          <div class="line3"></div>
+        </div>
+      </nav>
+    </div>
+    <div class="set-time">
+      <p class="today-date"></p>
+    </div>
+    <div class="awesome">
       <h1>Awesome Books</h1>
-    </header>
-    <main id="main">
       <div class="books"></div>
+    </div>
+    <section class="input-data">
+      <h3>Add A Book</h3>
       <form method="get">
         <input id="title" type="text" placeholder="Title" required><br><br>
         <input id="author" type="text" placeholder="Author" required><br><br>
         <button type="submit" id="add">Add</button>
       </form>
-    </main>
-    <script src="./index.js"></script>
-  </body>
+    </section>
+    <div class="container d-flex">
+      <section id="div-contact">
+        <h2 class="contact-info">Contact Information</h2>
+          <ul class="contactInfo d-none">
+              <li><span>Do you have any questions or you want to reach out to say 'Hello'?</span>You can reach out to us!</li>
+              <li><span>Our e-mail:</span> johndoe@gmail.com</li>
+              <li><span>Our phone number:</span> 004358645378</li>
+              <li><span>Our address:</span> Microverse</li>
+          </ul>
+      </section>
```

Comment:
- [ ] Please keep all the content in the center to follow the design. For now, it's toward the left and not looking good.

### Your design

![Screenshot from 2021-11-25 19-49-33](https://user-images.githubusercontent.com/22434660/143458569-bc19ab49-3e5e-45eb-86b0-7f5b1bc8b83a.png)

### Original design

![Screenshot from 2021-11-25 19-53-30](https://user-images.githubusercontent.com/22434660/143458518-4d4ab6f1-a6a6-47ef-a95d-645fca634876.png)